Makefile

```diff
@@ -11,16 +11,26 @@ build: ## Compile the Go package. This is the default.
 
 clean: ## Remove the .tools folder and binaries.
 	@rm -rf .tools
-	@rm terraform-provider-aap
+	@rm terraform-provider-aap || true
```

Comment:
À propos of nothing in particular, you can ignore results in Make with a minus in front of the production:
```
-@rm terraform-provider-aap
```

Reply:
Thanks for that!  I'll adopt and update.